02-express-tutorial/10-middleware-options.js

```diff
@@ -0,0 +1,38 @@
+const express = require ('express')
+const app = express()
+const logger = require('./logger')
+const authorize = require('./authorize')
+const morgan = require('morgan')
+
+
+
+//req => middleware => res
+
+//applies to relevant paths api/home/about/products
+//app.use('/api', logger)
+
+// applies to all paths 
+//app.use([logger,authorize]) //console.logs in order 
+
+app.use(morgan('tiny'))
+
```

Comment:
You could app.use[logger, morgan(tiny)] and don't need it in every route this way.  middleware is used by the app automatically when you execute app.use()